python-lib/ontology_tagger.py

```diff
@@ -0,0 +1,153 @@
+"""Main module to tag the documents"""
+
+from spacy_tokenizer import MultilingualTokenizer
+from dkulib_io_utils import generate_unique
+from spacy.matcher import PhraseMatcher
+from fastcore.utils import store_attr
+from typing import AnyStr, List
+import pandas as pd
+from time import perf_counter
+import logging
+from formatter_instanciator import FormatterInstanciator
+
+
+class Tagger:
+    def __init__(
+        self,
+        ontology_df: pd.DataFrame,
+        tag_column: AnyStr,
+        category_column: AnyStr,
+        keyword_column: AnyStr,
+        language: AnyStr,
+        lemmatization: bool,
+        case_insensitive: bool,
+        normalization: bool,
+        output_format: AnyStr,
```

Comment:
Perhaps move that as well to a keyword argument to tag_and_format?

Reply:
You're right !